src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -465,11 +465,18 @@ private int CheckAndAdjustPositionForSpeculativeAttribute(int position)
 
         #endregion Helpers for speculative binding
 
-        protected override IOperation GetOperationCore(SyntaxNode node, CancellationToken cancellationToken)
+        protected override IOperation GetOperationCore(IMethodSymbol method, CancellationToken cancellationToken)
         {
-            var csnode = (CSharpSyntaxNode)node;
-            CheckSyntaxNode(csnode);
-            return this.GetOperationWorker(csnode, GetOperationOptions.Lowest, cancellationToken);
+            var csmethod = (MethodSymbol)method;
+            CheckSymbol(csmethod);
+            var csbody = csmethod.GetBodySyntax();
+            CheckSyntaxNode(csbody);
```

Comment:
>CheckSyntaxNode(csbody); [](start = 12, length = 24)

I think consumer might be confused by an exception thrown by this helper because syntax is not an input for the GetOperation API.